### PR TITLE
support storage capacity for storage class

### PR DESCRIPTION
**Pull request by Hexilee**

### CODE REVIEW

File: pkg/driver/node.go [hunk 1]
```diff
@@ -127,10 +129,39 @@ func (d *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
...
+			}
+			if pair[0] == "subdir" {
+				subdir = path.Join("/", pair[1])
+			}
+		}
+		settings, err := d.juicefs.Settings(ctx, volumeID, secrets, volCtx, options)
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "get settings: %v", err)
+		}
+		output, err := d.juicefs.SetQuota(ctx, secrets, settings, path.Join(subdir, quotaPath), capacity)
```
Comment (zwwhdls) [inline pkg/driver/node.go:158]: get `subPath` from settings and `subdir` from settings.Options, it's trimSpaced.

**Comment by zwwhdls** [comment]
Also, add some e2e test about size limit please.

File: pkg/driver/node.go (hunk 1 above)
Comment (Hexilee) [reply pkg/driver/node.go:158]: Done

**Comment by Hexilee** [comment]
> Also, add some e2e test about size limit please.

Done

**Review by zwwhdls — APPROVED** [review]

**Pull request merged**